source/integrate-with-integration-environment/authenticate-your-user.html.md.erb

```diff
@@ -380,6 +384,7 @@ Now you’ve understood what’s in your ID token, you’ll need to validate it.
 1. Check the current time is after the time in the `iat` claim.
 1. If you set a `nonce` value in the request to the `/authorize` endpoint, check this matches the `nonce` value in the ID token.
 1. The `vot` claim must contain the credential trust level you asked for in the request to the `/authorize` endpoint. The `vot` claim will only contain the credential trust level, not the level of confidence, even if you make an identity request. For example, if you set the `vtr` parameter to `Cl.Cm.P2`, you must ensure the `vot` claim is equal to `Cl.Cm`.
+1. If you included `max_age` in the request to the `/authorize` endpoint, you must validate that `auth_time` is greater than or equal to the current time subtract the value of `max_age`. If false, you should reject the ID token and redirect the user to re-authenticate, by including `prompt=login` in the request to the `/authorize` endpoint.
```

Comment:
This won't work - as far as I'm aware we only use prompt=login for reauth. You'll want them to reauth with max_age again